_data/community.yml

```
@@ -210,7 +210,7 @@
     class: tile-medium text-dark
     style:
     - 'background-color: #fff'
-    - 'background-image: url(https://scontent-cdg2-1.xx.fbcdn.net/v/t1.0-9/72382200_108925870518281_822532927773474816_n.jpg?_nc_cat=104&_nc_oc=AQm_34GcGon5LEjPR87D_fj6jABRmeWAg_uJPZc-05lYuaDcFPNC5S7vHnrcJAw3Mi0&_nc_ht=scontent-cdg2-1.xx&oh=93f97a0080f7764e02c67e96c7254082&oe=5E46B1F9)'
+    - 'background-image: url(https://graph.facebook.com/Pôle-Gastronomique-107151704029031/picture?width=256)'
```

This one is giving me a 404 error in Chrome, I think this is the right one:

```suggestion
    - 'background-image: url(https://graph.facebook.com/107151704029031/picture?width=256)'
```